src/deferred.js

```diff
@@ -240,29 +242,29 @@ jQuery.extend({
 
 					// rejected_callbacks.disable
 					// fulfilled_callbacks.disable
-					tuples[ i ^ 1 ][ 2 ].disable,
```

Comment:
This looks like it was the only use of the xor operator in the whole codebase. I hope you're happy now.